### DISCUSSION TITLE
Preppers

### DISCUSSION THREAD

**Discussion by PersonHood** [WeAreMahsaAmini/FreeInternet#32] [General]
NOT AN ISSUE

Helpful info you may not have (tl;dr your stuff).
_____________________________

In the US there is a movement referred to as Preppers (search for it).

I am in the process of writing about MeshNets and Cantennas (hardware) using a computer like the RaspberryPi (search for each) powered by micro and mini FuelCells (search for a company in Latham, NY) and 12VDC deep discharge batteries with solar panel trickle chargers. Also a lot of 12VDC appliances and accessories (search RV and Boating electronics and electrical suppliers for ideas)

1st - this builds an Intra-net (LAN) with low leakage point to point connections ~5 miles line of sight, using any mobile phone or computer, locally standard coverage applies (no cantenna). Man in the middle attacks are minimized with Cantennas (and Woktennas). Off the grid power from 12VDC batteries. Stationary hotspots (Raspberry Pi computers) fill in signal gaps. May attach camera module for remote surveillence (Search for baby monitor projects using RaspberryPi or Adruino)

https://www.arduino.cc/

https://www.raspberrypi.com/

2nd avoid inserting SIMs in mobile phones, with reliance on 'carriers', vs VOIP and IP texting, and IP video chat. No Internet required but will work over inet if available. No radio noise from SIM (carrier data plans) with airplane mode turned on. Also longer battery life with airplane mode turned on.

3rd any one or more hotspots (tethered) connecting to the Internet enables all devices on the mesh-net to access the Internet. If techi enough set up each mesh-net as a LAN with LAN DHCP/DNS servers and one or more network bridges with NAT to the Internet. Tor/Orbot will encrypt traffic and exit node you in Norway, etc.

Connecrions to the Inet may include satelite phone, dish antennas, stealing (distant) public WiFi signals (with cantenna).

Learn Tor/Orbot and Orbot-friendly apps. 

Search my repositories for OpSec, a markdown (md) file with Android app info and web services info.

Good Luck!

**Comment by PersonHood** [comment]
ps

The network bridge may be configured as an incoming traffic firewall. 

Search GitHub for portspoof. 

Learn IP Tables. 

Set up a device in promiscuos mode.

https://en.wikipedia.org/wiki/Promiscuous_mode

**Comment by PersonHood** [comment]
ps

The network bridge may be configured as an incoming traffic firewall. 

Search GitHub for portspoof. 

Learn IP Tables. 

Set up a device in promiscuos mode.

https://en.wikipedia.org/wiki/Promiscuous_mode

**Comment by PersonHood** [comment]
ps

The network bridge may be configured as an incoming traffic firewall. 

Search GitHub for portspoof. 

Learn IP Tables. 

Set up a device in promiscuos mode.

https://en.wikipedia.org/wiki/Promiscuous_mode

**Comment by PersonHood** [comment]
ps

The network bridge may be configured as an incoming traffic firewall. 

Search GitHub for portspoof. 

Learn IP Tables. 

Set up a device in promiscuos mode.

https://en.wikipedia.org/wiki/Promiscuous_mode

**Comment by theStratus** [comment]
These ideas may be merged. 
https://github.com/WeAreMahsaAmini/Core/discussions/23#discussioncomment-3775866
